Detectors/Upgrade/ITS3/simulation/include/ITS3Simulation/Detector.h

```diff
@@ -0,0 +1,382 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file Detector.h
+/// \brief Definition of the Detector class
+
+#ifndef ALICEO2_ITS3_DETECTOR_H_
+#define ALICEO2_ITS3_DETECTOR_H_
+
+#include <vector>                             // for vector
+#include "DetectorsBase/GeometryManager.h"    // for getSensID
+#include "DetectorsBase/Detector.h"           // for Detector
+#include "DetectorsCommonDataFormats/DetID.h" // for Detector
+#include "ITSMFTSimulation/Hit.h"             // for Hit
+#include "Rtypes.h"                           // for Int_t, Double_t, Float_t, Bool_t, etc
+#include "TArrayD.h"                          // for TArrayD
```

Comment:
is this needed?

Reply:
`TArrayD.h`? not at all, here can be really removed. See also previous comment.